posthog/__init__.py

```diff
@@ -454,6 +455,10 @@ def _proxy(method, *args, **kwargs):
             disabled=disabled,
             disable_geoip=disable_geoip,
             feature_flags_request_timeout_seconds=feature_flags_request_timeout_seconds,
+            # TODO: Currently this monitoring begins only when the Client is initialised (which happens when you do something with the SDK)
+            # This kind of initialisation is very annoying for exception capture. We need to figure out a way around this,
+            # or deprecate this proxy option fully (it's already in the process of deprecation, no new clients should be using this method since like 5-6 months)
```

Comment:
> which happens when you do something with the SDK

that means someone has to do `posthog.capture` and only then it works?
assuming that the auto capture exception also calls `posthog.capture`, should it work as it is?

Reply:
sadly no, because at that point its too late, we need to initialise so the hooks exist before any exception occurs (capture is called after the exception). I'll make sure all docs don't mention the proxy version and only the initialisation version which has none of these problems